core/build.gradle.kts

```diff
@@ -0,0 +1,90 @@
+val kotlinVersion: String by project
+
+plugins {
+    kotlin("multiplatform") version "1.3.60-eap-76"
+}
+
+kotlin {
+    /*infra {
+        target("macosX64")
+        target("linuxX64")
+        target("mingwX64")
+    }*/
```

Comment:
infra should work now, let's declare native targets with it.

Reply:
`infra` isn't seen in this file.

Reply:
Moreover I need to do some other settings in targets block. s this possible with infra?